### PR TITLE
refact: fix issues msg by tags

### PR DESCRIPTION
**Pull request by kaioken**

### CODE REVIEW

File: src/modules/messages/index.ts [hunk 1]
```diff
@@ -95,13 +97,17 @@ export class Messages {
     orderBy: Array<OrderByMessage>,
     search: string,
     first: number,
-    page: number
+    page: number,
+    hasTags?: HasTagWhereHasConditions,
```
Comment (jerlyrosa) [inline src/modules/messages/index.ts:101]: Change: use the WhereCondition interface

File: src/types/messages.ts [hunk 2]
```diff
@@ -68,6 +68,16 @@ export interface HasAppModuleMessageWhereConditions {
   operator: string;
   value: string;
 }
+export interface HasTagWhereHasConditions {
```
Comment (jerlyrosa) [inline src/types/messages.ts:71]: we already have a declared interface that you can also use for this, it's called **WhereCondition**.

File: test/message.test.ts [hunk 3]
```diff
@@ -74,6 +74,68 @@ describe('Test the Social Messages', () => {
         //const { data } = recentMessages;
```
Comment (jerlyrosa) [inline test/message.test.ts:74]: remove

File: test/message.test.ts [hunk 4]
```diff
@@ -74,6 +74,68 @@ describe('Test the Social Messages', () => {
...
+            {} as HasAppModuleMessageWhereConditions,
+            [{ column: 'CREATED_AT', order: 'DESC' }],
+            '',
+            25,
+            1,
+            {} as HasTagWhereHasConditions,
+            {column: 'VERB', operator: 'EQ', value: 'post2'} as HasTypeWhereHasConditions
+        );
+        expect(recentMessages).toBeDefined();
+        //console.log(recentMessages);
```
Comment (jerlyrosa) [inline test/message.test.ts:107]: Remove these comments

File: test/message.test.ts [hunk 5]
```diff
@@ -74,6 +74,68 @@ describe('Test the Social Messages', () => {
...
+            {} as WhereCondition,
+            {} as HasAppModuleMessageWhereConditions,
+            [{ column: 'CREATED_AT', order: 'DESC' }],
+            '',
+            25,
+            1,
+            {column: 'SLUG', operator: 'EQ', value: 'post2'} as HasTagWhereHasConditions,
+        );
+        expect(recentMessages).toBeDefined();
+        //console.log(recentMessages);
```
Comment (jerlyrosa) [inline test/message.test.ts:135]: same here

**Review by jerlyrosa — CHANGES_REQUESTED** [review]

**Review by AudryLopez — APPROVED** [review]

**Review by jerlyrosa — APPROVED** [review]

**Pull request merged**